var/spack/repos/builtin/packages/freesurfer/package.py

```diff
@@ -14,11 +18,11 @@ class Freesurfer(Package):
 
     # A license is required, but is free to obtain.
     license_required = True
-    license_files = ["./license.txt"]
```

Comment:
the documentation still suggests the license should be license.txt in the prefix of this package.

Reply:
the thing is when it was called license.txt it crashed because it did not find it. I didn't dig into it too much

Reply:
Is freesurfer a licensed product? (no pun intended)
Can it be replaced with a dummy?

Reply:
It's a licensed (free) product.